app/views/repositories/_navigation.html.erb

```diff
@@ -33,21 +33,33 @@ See doc/COPYRIGHT.rdoc for more details.
 
 <% content_for :action_menu_specific do %>
   <!--[if gt IE 8]><!-->
```

Comment:
@ulferts not directly related to this task, do we still need this conditional comment?